### DISCUSSION TITLE
Difference in 0.2 and 0.1?

### DISCUSSION THREAD

**Discussion by DariBer** [frntc/SIDKick-pico#66] [Q&A]
I'm trying to find some info to see what the see what the differences are between v0.2 and v0.1 PCBs.

**Comment by frntc** [comment]
Mainly two changes: reset handling and enabling the mouse-trigger-option (setting variable voltage thresholds for reading the mouse x/y-values)

**Reply by DariBer** [reply]
If I have 0.1 boards, would you recommend me to not use them and order 0.2 boards?

**Reply by frntc** [reply]
The only situation where you would experience differences is when you launch very old cartridge games (a handful, not many), which initialize the SID only once just a few cycles after reset. There's also a way to modify the 0.1 PCBs, but not sure it's worth it.

**Reply by gsargent70** [reply]
I wouldn't mind knowing how to modify the 0.1 PCBs as I am getting ready to build four for myself and a friend.  I don't forsee either of us ever using the DAC so don't really want to order new boards.  We will be running the audio straight through the C64 like you would with a vintage SID.  I appreciate the info and the project!

**Reply by frntc** [reply]
There are two options:
1 (easiest when you build the SKpico): do not connect pin 30 of the Pico (e.g. leave pin out in the pin header), and on the SKpico-PCB you connect pin 29 and 30.

2 (easier when you don’t want 1, e.g. when it’s already built and soldered together): flip the diode as on the image and connect its anode to pin 29.

<img width="579" alt="image" src="https://github.com/user-attachments/assets/3e20a4f9-6b2c-45da-9d56-a20d62b9734c">

**Reply by modeler** [reply]
Sorry to bump an old thread, but I have an observation. I performed the Number 2 mod on a previously built board (thanks for the detailed diagram by the way!) and assumed you'd use the 0.2 firmware image with device once this mod was in place.

I got some very crackly and stuttering output from that, however the 0.1 firmware works fine and the issue with missing audio with some cartridge software was indeed fixed. So I would deduce that there are still some differences between the 0.2 and modified 0.1 board, and the mod shown here fixes that specific design issue in a different way. Please correct me if I'm wrong!

**Reply by frntc** [reply]
In fact the only difference between 0.1 and 0.2 (no-DAC version of course) is the mode that the image above shows. For testing you could connect the GPIO via a large pullup resistor to 3.3V (but that should not be necessary).
